util-hadoop/src/test/java/com/google/cloud/hadoop/util/HadoopCredentialConfigurationTest.java

```diff
@@ -250,6 +250,10 @@ private static String getStringPath(String resource) throws Exception {
   }
 
   private static Path getPath(String resource) throws Exception {
-    return Paths.get(Resources.getResource(resource).getFile());
+    String filePath = Resources.getResource(resource).getFile();
+    return (System.getProperty("os.name").toLowerCase().contains("win")
+            && filePath.charAt(0) == '/')
```

Comment:
Use `filePath.startsWith('/')` instead of `filePath.charAt(0) == '/'`.

Reply:
Fixed.